src/plugins/csvstorage/README.md

```diff
@@ -17,12 +17,21 @@ Tells the plugin what delimiter is used in the file.
 The default delimiter is `;` and will be used if `delimiter` is not set.
 
 `useheader`
-Tells the plugin to use the first line as a header if it's set to anything but "0". The columns will get the coresponding names.
-If useheader is not set, the columns get namend #0,#1,...
+Tells the plugin to use the first line as a header if it's set to "1". The columns will get the coresponding names.
+Ignore the first line if it's set to "-1" or treat the first line as a record if it's set to "0". 
```

Comment:
can you use a different key or change useheader to an "enum". Using -1 is cryptic.